### PR TITLE
WEBEU2 - Responsive Web Design I - Isaac Aderogba 🇮🇪

### PR DESCRIPTION
**Pull request by IsaacAderogba**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,121 @@
...
+
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/styles.css">
```
Comment (maximesalomon) [inline index.html:12]: You are missing the meta tag `content="width=device-width, initial-scale=1"`

Comment (maximesalomon) [reply index.html:12]: Can you explain what does it do?

File: index.html [hunk 2]
```diff
@@ -1,16 +1,121 @@
...
+        <a href="#" class="nav-item">Contact</a>
+      </nav>
+      <div class="nav-logo">
+        <a href="index.html">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
+      </div>
+    </header>
+
+    <section class="hero">
```
Comment (maximesalomon) [inline index.html:36]: Either I'll put the nav and the first section in the header or just the first section. But the first section can't be let alone between the `header` and `main`

File: index.html [hunk 3]
```diff
@@ -1,16 +1,121 @@
...
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+      </section>
+    </main>
+
+    <hr>
+    <section class="contact">
+      
+      <h2>Contact</h2>
+      <p>123 Way 456 Street<br>
```
Comment (maximesalomon) [inline index.html:109]: you can use the`<adress>` here

File: services.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! Services</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/styles.css">
+
```
Comment (maximesalomon) [inline services.html:12]: Similar feedbacks to index.html

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+
+.nav-item {
+	color: grey;
+	text-decoration: none;
+}
+
+
+
+
+/* Media Query for Page, Header and Nav elements*/
```
Comment (maximesalomon) [inline css/styles.css:113]: Great use of comments all over your file. Great job!

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+@media(max-width: 500px) {
+	:root {
+		font-size: 20px;
+	}
+}
+
+/* Index Page */
+
+.container {
+	margin: 1rem auto;
```
Comment (maximesalomon) [inline css/styles.css:82]: rem 🔥

**Review by maximesalomon — COMMENTED** [review]
Great job on this project Isaac! Seems like you have a pretty good understanding of media queries and how to implement them. Pay a bit more attention to your HTML markup.

Overall amazing job 🔥

File: index.html (hunk 1 above)
Comment (IsaacAderogba) [reply index.html:12]: From what I understood, it controls the window in which the content is seen. I believe that manipulating the `initial-scale` creates a sort of zoom-in or zoom-out effect! I must have omitted it in the submission, so will update

File: index.html (hunk 2 above)
Comment (IsaacAderogba) [reply index.html:36]: Okay got it,

Could the same logic be made for the footer, in that the section "contact" class should sit inside either the main or the footer element

**Pull request merged**